app/javascript/hyacinth_v1/components/batch_imports/BatchImportNew.js

```diff
@@ -108,7 +108,7 @@ function BatchImportNew() {
               sm={12}
               value={priority}
               onChange={setPriority}
-              options={priorities.map(p => ({ value: p, label: `Priority: ${startCase(p)}` }))}
+              options={priorities.map(p => ({ value: p.toUpperCase(), label: `Priority: ${startCase(p)}` }))}
```

Comment:
Rather than upcase this `p` value, it might be better to change `const priorities = ['low', 'medium', 'high'];` earlier in the file to `const priorities = ['LOW', 'MEDIUM', 'HIGH'];`

Reply:
And I think we may also need to change `const [priority, setPriority] = useState('medium');` to `const [priority, setPriority] = useState('MEDIUM');`